build_debian.sh

```diff
@@ -318,7 +318,7 @@ sudo LANG=C chroot $FILESYSTEM_ROOT useradd -G sudo,docker $USERNAME -c "$DEFAUL
 echo "$USERNAME:$PASSWORD" | sudo LANG=C chroot $FILESYSTEM_ROOT chpasswd
 
 ## Create redis group
-sudo LANG=C chroot $FILESYSTEM_ROOT groupadd -f redis
+sudo LANG=C chroot $FILESYSTEM_ROOT groupadd -f -g $REDIS_USER_GID redis
```

Comment:
>$REDIS_USER_GID [](http://example.com/codeflow?start=51&length=15)

Is there possibility that there is already other group occupied this gid? Will this command fail? or just force reuse (which is not expected)?

Reply:
Fixed